### DISCUSSION TITLE
What I hate about home assistant.

### DISCUSSION THREAD

**Discussion by Marc19550831** [community/community#136414] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

In the read.me file it tells you to add a lines of yaml, but it doesn't tell you where to add it or how to add it. It just says to add it. So, my first is to add it to config.yaml usually makes HA mad it tells me to remove it, then it's hit and miss. I try to add it to a blank dashboard, that never works... I wish if you going to give instructions, make sure they are complete, not assume everyone knows everything. It been a strugle to learn what I have learned because everyone assumes you should know it.

**Comment by LiteBrite82** [comment]
Hi USERNAME,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
